sql_generators/glean_usage/__init__.py

```diff
@@ -144,10 +136,5 @@ def generate(project_id, output_dir, parallelism, exclude, only, app_name):
         for table in GLEAN_TABLES
     ]
 
-    with Pool(parallelism) as pool:
-        pool.starmap(run_generate, generate_per_app_id + generate_per_app)
-
-
-def run_generate(func, params):
-    """Use in `generate()` for generating glean datasets in parallel."""
-    func(params)
+    with ProcessingPool(parallelism) as pool:
+        pool.map(lambda f: f[0](f[1]), generate_per_app_id + generate_per_app)
```

Comment:
Use `pathos` for parallel processing. The standard `multiprocessing` module has problems with executing functions in parallel that are not part of a package.